src/utils/response/responseHandlers.ts

```diff
@@ -118,6 +118,16 @@ export async function serverCommunication<ResourceName extends ResourceNames>(
   }
 }
 
+export async function plainObject<ResourceName extends ResourceNames>(
+  promise: RequestPromise<ResourceName>
+): Promise<{ Results: Entity<ResourceName> }> {
+  const response = (await toApiResponse<ResourceName>(promise)) as unknown as ApiResponseOnSuccess<ResourceName, Entity<ResourceName>>;
```

Comment:
The `as unknown` cast seems suspicious

Reply:
it works same way as for serverCommunication